ixdjango/docker_settings.py

```diff
@@ -42,11 +42,14 @@
 USE_X_FORWARDED_HOST = True
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
+# Allowed hosts
+try:
+    ALLOWED_HOSTS = os.environ.get('SITE_DOMAIN').split('|')
+except AttributeError:
```

Comment:
Better to use os.environ[badger] and catch KeyError. AttributeError is not a great one to catch because you could be raising it for lots of reasons that eat a real programming error.